.drone.yml

```diff
@@ -72,6 +79,13 @@ steps:
       repo: quay.io/${DRONE_REPO_NAMESPACE}/performance-addon-operator-registry
       auto_tag: true
 
+  - name: validate-registry-image
+    image: quay.io/${DRONE_REPO_NAMESPACE}/performance-addon-operator-registry:${DRONE_TAG}
+    commands:
+      - sqlite3 /registry/bundles.db "select * from package" | grep "performance-addon-operator|${DRONE_SEMVER_SHORT}"
+      - sqlite3 /registry/bundles.db "select * from channel" | grep "${DRONE_SEMVER_SHORT}|performance-addon-operator|performance-addon-operator.v${DRONE_SEMVER_SHORT}"
+      - sqlite3 /registry/bundles.db "select * from operatorbundle" | grep "\"image\":\"quay.io/${DRONE_REPO_NAMESPACE}/performance-addon-operator:${DRONE_TAG}\""
```

Comment:
very nice, a bit too low level. Is this supported by the OLM folks?

Reply:
answered on #196